### DISCUSSION TITLE
split multichannel audio file to single audio files

### DISCUSSION THREAD

**Discussion by inst5** [cables-gl/cables_docs#221] [Answers & Questions] [answered]
Hi,
if I understood it right, cable can only play mono audio files.
If I have an OPUS audio file with lets say 8 channels of audio. Loaded with FileInput.
Is there a was to split the 8ch filen into 8 seperate mono files to play them back or process them with audiobuffer -> audiobufferplayer ?

thanks!

**Comment by tastez** [comment, marked as answer]
hi @inst5 , cables.gl does support stereo playback of files! 
As far as splitting a multichannel OPUS audio file into mono channels this isn't possible with any current OP but if it is supported with WebAudio then you can make a customized op for sure. Feel free to find some examples and bring em to our meetups!
